### PR TITLE
WIP: Upgrade Symfony 4

### PR DESCRIPTION
**Pull request by ohvitorino**
This PR updates the framework to use Symfony 4

### CODE REVIEW

**Comment by ohvitorino** [comment]
As the title says: Work In Progress 🙃

File: app/config/config.yml [hunk 1]
```diff
@@ -180,16 +180,16 @@ swiftmailer:
...
-jms_translation:
-  configs:
-    app:
-      dirs: ["%kernel.root_dir%", "%kernel.root_dir%/../src"]
-      output_dir: "%kernel.root_dir%/Resources/translations"
-      ignored_domains: [routes]
-      excluded_names: ["*TestCase.php", "*Test.php", "DefaultMenuListener.php"]
-      excluded_dirs: [cache, data, logs]
-      extractors: [jms_i18n_routing]
+## Translation configuration
```
Comment (jonasdekeukelaere) [inline app/config/config.yml]: just remove this? as far as I know this isn't used anywhere?

**Comment by jonasdekeukelaere** [comment]
Test vinden dit precies nog niet zo leuk

File: app/config/config.yml (hunk 1 above)
Comment (ohvitorino) [reply app/config/config.yml]: I'm confused with your question marks. AFAIK none of us uses the bundle anymore, there's not even support for Symfony 4 yet as far as I could tell. I'll just remove the comments.

**Comment by ohvitorino** [comment]
There's some stuff that changed, so travis config still needs to be fixed.

File: app/config/config.yml (hunk 1 above)
Comment (jonasdekeukelaere) [reply app/config/config.yml]: Nvm, I saw the first comment line and though you just put it in comments, but you actually deleted it, so all ok!

**Comment by jonasdekeukelaere** [comment]
@ohvitorino any status update on this?

**Comment by ohvitorino** [comment]
@jonasdekeukelaere Nop. I didn't have time to work on this in the last weeks. I was actually considering to setup Symfony 4 from scratch and them include our stuff using flex.

**Comment by jonasdekeukelaere** [comment]
Isn't that going to be more work? Because we did loads of work in the Framework?

**Comment by ohvitorino** [comment]
Maybe. We have to discuss this I think.

**Comment by jonasdekeukelaere** [comment]
Symfony 4 will be a rewrite

**Pull request closed without merging**